frontend/src/ches/MergeForm.jsx

```diff
@@ -655,25 +696,35 @@ class MergeForm extends Component {
           let form = this.state.form;
           let contexts = [];
           excel.data.forEach(d => {
-            let r = {to: [], cc: [], bcc: [], context: {}};
+            let r = {to: [], cc: [], bcc: [], tag: '', delayTS: undefined, delayTSDisplay: undefined, context: {}};
```

Comment:
Should tag also be undefined instead of empty string? A tag is not a required field for CHES, so it could actually not even exist and it'd still be processable.

Reply:
delayTS can't be anything other than undefined or an int.   Whereas tag can be '' and it won't matter. i'll make the change anyway.